waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/parse/ASTConverter.java

```diff
@@ -30,12 +30,12 @@
 import java.util.List;
 import java.util.Map;
 
+import org.apache.commons.lang3.tuple.ImmutablePair;
```

Comment:
this class gets pulled it via hive-exec jar, not great but I don't want to make a too involved change now (We shouldn't use hive-exec but hive-exec-core.jar etc.... out of scope).

Reply:
Sure, using another Pair that is already on the classpath sounds like the quickest fix for now. Later we can decide if we want to introduce a new class or use something else.